src/components/job/FilterForm.vue

```diff
@@ -0,0 +1,162 @@
+          v-text-field(
+            label="Keywords",
+            :value="keywords.join(' ')"
+            color="white",
+            clearable,
+            @input="$emit("update:keywords", $event.split(' '))"
+          )
+    v-layout(row, justify-center)
+      v-flex(sm3, d-flex)
+        v-select(
+            label="Job Type",
+            :items="jobTypeList",
+            :value="jobType",
+            clearable,
+            color="white",
+            :menu-props="{'content-class':'filter-form-menu'}",
+            dark,
+            dense,
+            @input="$emit('update:jobType', $event ? $event : null)")
+      // for when girder's api supports this sort of filtering
+      //
+        v-flex(sm6, d-flex)
+          v-layout(row)
+            v-flex(sm6)
+              v-menu(
+                ref="fromDateMenu",
+                :close-on-content-click="false",
+                :nudge-right="40",
+                v-model="fromDateMenu",
+                lazy,
+                transition="scale-transition",
+                offset-y,
+                full-width
+              )
+                v-text-field(
+                  slot="activator",
+                  :value="fromDate",
+                  label="Last Update (from)",
+                  prepend-icon="event",
+                  clearable,
+                  color="white",
+                  readonly
+                )
+                v-date-picker(
+                  :value="fromDate",
+                  no-title,
+                  dark,
+                  @input="$emit('update:fromDate', $event); fromDateMenu = false"
+                )
+            v-flex(sm6)
+              v-menu(
+                ref="toDateMenu",
+                :close-on-content-click="false",
+                :nudge-right="40",
+                v-model="toDateMenu",
+                lazy,
+                transition="scale-transition",
+                offset-y,
+                full-width
+              )
+                v-text-field(
+                  slot="activator",
+                  :value="toDate",
+                  label="Last Update (to)",
+                  prepend-icon="arrow_forward",
+                  clearable,
+                  color="white",
+                  readonly
+                )
+                v-date-picker(
+                  :value="toDate",
+                  no-title,
+                  dark,
+                  @input="$emit('update:toDate', $event); toDateMenu = false"
+                )
+      v-flex(sm3, d-flex)
+        v-select(
+            label="Status",
+            :items="statusItemList",
+            :value="status",
+            clearable,
+            color="white",
+            :menu-props="{'content-class':'filter-form-menu'}",
+            dark,
+            dense,
+            @input="$emit('update:status', $event ? $event : null)")
+</template>
+
+<style lang="scss">
+.filter-form-menu .v-list__tile--active.theme--light {
```

Comment:
Could we give the global style a more specific name, like `.gwc-job-filter-form-menu` ?

Reply:
Yes, I am thinking `girder-job-filter-form-menu`.